source/guides/multisig/modifying-a-multisig-account-min-approval.rst

```diff
@@ -37,8 +37,36 @@ Prerequisites
 
 - Complete :doc:`converting an account to multisig <creating-a-multisig-account>` guide.
 
+************************************
+Method #01: Using the Desktop Wallet
+************************************
+
+1. Log in to an account that is a cosignatory of the account you want to modify. Either Alice or Bob's account.
+
+2. Click on the “**Multisig**” tab on the left-side of the wallet.
+
+3. Select the multisig wallet you want to modify from the dropdown menu from the top field. This will convert the “**Operation Type**” to “**Modifying account multisig properties**”.
+
+.. figure:: ../../resources/images/screenshots/modify-multisig-1.gif
+    :align: center
+    :width: 800px
+
+4. Select the new “**Min. Approval**” number. In this case, we want to change it to "**2**". Click “**Send**”. Review the information on the popup. Enter your wallet password and click “**Confirm**”.
+
+.. figure:: ../../resources/images/screenshots/modify-multisig-2.gif
+    :align: center
+    :width: 800px
+
+5. Navigate to the “**Home**” page. Click on the “**partial**” transactions. Click on the pending transaction. Enter your password and click “**Confirm**”.
```

Comment:
The cosignatory that announces the transaction is not required to cosign it again. At least minApproval cosignatories must approve the transaction so that this is included in a block.